userguide/network.rst

```diff
@@ -737,6 +737,18 @@ More information about this command can be found at
 `systat(1) <https://www.freebsd.org/cgi/man.cgi?query=systat>`__.
 
 
+.. _Network Summary:
+
+Network Summary
+---------------
+
+:menuselection:`Network --> Network Summary`
+shows a quick summary of the addressing information of every
+configured interface. For each interface name, the configured IPv4 and
+IPv6 addresses, DNS servers, and default gateway are displayed. It
+also displays nameservers.
```

Comment:
This last sentence doesn't make sense.  DNS servers *are* nameservers.  Should also put DNS default gateway with the other network information:
"IPv6 addresses, default gateway, and DNS nameservers."